src/Magnum/Audio/Playable.h

```diff
@@ -0,0 +1,178 @@
+@ref SceneGraph::Object::setClean() is called, which is done in
+@ref SceneGraph::Camera::draw() for example.
+
+To manage multiple Playables at once, use @ref PlayableGroup.
+
+-   @ref Playable2D
+-   @ref Playable3D
+
+@see @ref Source, @ref Listener, @ref PlayableGroup
+*/
+template<UnsignedInt dimensions> class Playable: public SceneGraph::AbstractGroupedFeature<dimensions, Playable<dimensions>, Float> {
+    friend PlayableGroup<dimensions>;
+
+    public:
+
+        /**
+         * @brief Constructor
+         */
+        explicit Playable(SceneGraph::AbstractObject<dimensions, Float>& object, Source& source, PlayableGroup<dimensions>* group = nullptr):
+            SceneGraph::AbstractGroupedFeature<dimensions, Playable<dimensions>, Float>{object, group},
+            _playableGroup{group},
+            _fwd{},
+            _gain{source.gain()},
+            _source{source}
+        {
+            SceneGraph::AbstractFeature<dimensions, Float>::setCachedTransformations(SceneGraph::CachedTransformation::Absolute);
+            _fwd[dimensions - 1] = 1;
+        }
+
+        /** @brief Source which is managed by this feature */
+        Source& source() {
+            return _source;
+        }
+
+        /**
+         * @brief Set source to manage
+         * @return Reference to self (for method chaining)
+         */
+        Playable& setSource(Source& source) {
+            _source = source;
+            Playable<dimensions>::object().setDirty();
+            return *this;
+        }
+
+        /** @brief Gain */
+        Float gain() const {
+            return _gain;
+        }
+
+        /**
+         * @brief Set gain of the playable and source respecting the PlayableGroups gain
+         *
+         * The sources gain is computed as `sourceGain = playableGain*groupGain`.
+         * @see @ref PlayableGroup::setGain(), @ref Source::setGain()
+         */
+        Playable& setGain(const Float gain) {
+            _gain = gain;
+            cleanGain();
+            return *this;
+        }
+
+    private:
+        void clean(const MatrixTypeFor<dimensions, Float>& absoluteTransformationMatrix) override {
+            Vector3 position = Vector3::pad(absoluteTransformationMatrix.translation(), 0);
+            if(_playableGroup) {
+                position = _playableGroup->soundTransformation()*position;
+            }
+            _source.setPosition(position);
+            _source.setDirection(Vector3::pad(absoluteTransformationMatrix.rotation()*_fwd));
+
+            // TODO: velocity
+        }
+
+        /* Update the gain of the underlying source to reflect changes in _group and/or _gain.
+           Called in Playable::setGain() and PlayableGroup::setGain() */
+        void cleanGain() {
+            if(_playableGroup) {
+                _source.setGain(_gain*_playableGroup->gain());
+            } else {
+                _source.setGain(_gain);
+            }
+        }
+
+        PlayableGroup<dimensions>* _playableGroup;
```

Comment:
Better solution might be to create `playables()` function that just `static_cast`s value returned from `AbstractGroupedFeature::group()` -- i.e., no need to store another variable. See `SceneGraph::Animable::animables()` for an implementation. 

This would break when the user would set the group by calling `AbstractGroupedFeature::setGroup()`.